ansible/roles/accumulo/tasks/main.yml

```diff
@@ -61,3 +61,12 @@
     creates: "{{ accumulo_home }}/lib/native/libaccumulo.so"
 - name: "Create accumulo log dir"
   file: path={{ worker_data_dirs[0] }}/logs/accumulo state=directory
+- name: "Enable debug logging in Accumulo 2.0"
+  replace: 
+    path: "{{ accumulo_home }}/conf/{{ item }}"
+    regexp: '^log4j.rootLogger.*'
+    replace: "log4j.rootLogger=DEBUG, file"
```

Comment:
When I run accumulo test I usually add a line like the following to `log4j-service.properties`

```
log4j.logger.org.apache.accumulo=DEBUG
```

This only enables debug logging for Accumulo code.  The change `log4j.rootLogger=DEBUG, file` will enable debug logging for everything like Hadoop and Zookeeper.  I am not sure how voluminous or useful this is.

Reply:
https://github.com/apache/fluo-muchos/blob/c9a87f458972f93ecedd603662924ca0afd632fd/ansible/roles/accumulo/templates/log4j-service.properties#L34
@keith-turner Hi, Since the line was already there in `log4j-service.properties`, I was under the impression that #263 was to print all DEBUG level messages on Accumulo logs and so modified `log4j.rootLogger=DEBUG, file`. Let me know if there is further change required?

Reply:
@karthick-rn  it looks like I made this change in 496d73f4c7cc9f2c4752226c6a3df297decce355.  For the testing use case I had, this existing change already satisfies that.  I completely forgot about that change and did not close #263 as I should have.  I don't think any additional changes are needed.   I am so sorry for wasting your time!

Reply:
@keith-turner No worries, I understand. Do you want me to submit a new PR for the `hadoop-metrics2.properties` that is missing in `/hadoop-ha/templates`?

Reply:
@karthick-rn You could probably just change the title of this PR and repurpose it for that change. :smiley_cat:

Reply:
Sounds good. I'll do it that way. Thanks @ctubbsii